kornia/feature/defmo.py

```diff
@@ -0,0 +1,112 @@
+from typing import Dict
+
+import torch
+import torch.nn as nn
+import torchvision.models
+
+urls: Dict[str, str] = {}
+urls["defmo_encoder"] = "http://ptak.felk.cvut.cz/personal/rozumden/defmo_saved_models/encoder_best.pt"
+urls["defmo_rendering"] = "http://ptak.felk.cvut.cz/personal/rozumden/defmo_saved_models/rendering_best.pt"
+
+
+class EncoderDeFMO(nn.Module):
+    def __init__(self):
+        super(EncoderDeFMO, self).__init__()
+        model = torchvision.models.resnet50(pretrained=True)
+        modelc1 = nn.Sequential(*list(model.children())[:3])
+        modelc2 = nn.Sequential(*list(model.children())[4:8])
+        pretrained_weights = modelc1[0].weight
+        modelc1[0] = nn.Conv2d(6, 64, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3), bias=False)
+        modelc1[0].weight.data[:, :3, :, :] = nn.Parameter(pretrained_weights)
+        modelc1[0].weight.data[:, 3:, :, :] = nn.Parameter(pretrained_weights)
+        self.net = nn.Sequential(modelc1, modelc2)
```

Comment:
I do not get it. Why split and merge again?

```python
pretrained_weights = model[0].weight
model[0] = nn.Conv2d(6, 64, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3), bias=False)
model[0].weight.data[:, :3, :, :] = nn.Parameter(pretrained_weights)
model[0].weight.data[:, 3:, :, :] = nn.Parameter(pretrained_weights)
```

Will this be not working?

Reply:
Agree

Reply:
No, it's not the same because I merge [:3] and [4:8] parts, omitting one MaxPooling layer [4].

Reply:
gotcha :+1: